src/HolographOperator.sol

```diff
@@ -497,10 +497,7 @@ contract HolographOperator is Admin, Initializable, HolographOperatorInterface {
    * @dev This function is restricted for use by Holograph Messaging Module only
    */
   function crossChainMessage(bytes calldata bridgeInRequestPayload) external payable {
-    require(
-      msg.sender == address(_messagingModule()) || msg.sender == 0x777C19834a1A2FF6353a1E9cfb7C799ed7943a11,
```

Comment:
We had identified this and will deploy to remove the V1 module. Thanks for opening the PR @ACC01ADE

Reply:
I see the fix is being applied via the `feature/holograph-operator-update` branch. Closing this PR.